### DISCUSSION TITLE
Account creation doesn't work

### DISCUSSION THREAD

**Discussion by kroeckx** [electricitymaps/electricitymaps-contrib#6274] [General]
Hi,

I'm trying to create a free personal account but it doesn't work. When I fill in all the things, I get a dialog box saying "Network error" for a few seconds. I seem to be getting a 400 http error from for https://api.blobr.app/blobr/organizations/org_b6vkd0bullrezyg1/users.

I've tried to contact them about 2 weeks ago without any reply.

**Comment by kroeckx** [comment]
The problem seems to be related to the password. I'm not sure what the problem is, but I've made the password shorter and have less special characters and was then able to sign up.

**Reply by madsnedergaard** [reply]
That is strange indeed, but I am happy you managed to get it working :)
